sopel/test_tools.py

```diff
@@ -17,11 +17,10 @@
 # Copyright 2013, Ari Koivula, <ari@koivu.la>
 # Copyright 2019, Florian Strzelecki <florian.strzelecki@gmail.com>
 # Licensed under the Eiffel Forum License 2.
-from __future__ import absolute_import, division, print_function, unicode_literals
+from __future__ import generator_stop
 
 import os
 import re
-import sys
 import tempfile
 
 try:
```

Comment:
Stupid proximity limits on line notes prevent me from providing a full suggestion, but this nonsense can be dropped in favor of a straight `import configparser` above, as it was in `sopel/config/__init__.py`.